src/pages/Home/index.tsx

```diff
@@ -1,9 +1,94 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { useState, useContext, useEffect } from 'react'
+import ColoredPokemonsContext from '../../context/ColoredPokemonsContext'
+import { useQuery } from '@tanstack/react-query'
+import { getPokemonList } from '../../helpers/tanStackQuery'
+import axios from 'axios'
+
+import FilterBar from '../../components/Home/FilterBar'
+import Content from '../../components/Home/Content'
+import TotalBar from '../../components/Home/TotalBar'
+import Pagination from '../../components/Home/Pagination'
+
+import { HomeWrap } from './style'
+
 const Home = () => {
-  return (
-    <>
-      <h2>Home</h2>
-    </>
-  );
-};
-
-export default Home;
+    const coloredPokemonsCtx = useContext(ColoredPokemonsContext)
+    const [currentPage, setCurrentPage] = useState<number>(1)
+    const [selectedColor, setSelectedColor] = useState<number>(0)
+
+    //Calling helper function which is enabling tanstack-query pagination and filter by color functionality
+    const { isError, isLoading, data, dataUpdatedAt } = useQuery({
```

Comment:
Ensure you are fetching data only once once the app is loaded, this is required for all GET requests.